src/main/resources/static/templates/XSSWithNullBytesImgTagAttribute/LEVEL_1/XSS.js

```diff
@@ -1,15 +1,24 @@
-function addingEventListenerToLoadImageButton() {
-  document.getElementById("loadImage").addEventListener("click", function () {
-    let url = getUrlForVulnerabilityLevel();
-    doGetAjaxCall(
-      appendResponseCallback,
-      url + "?value=images/" + document.getElementById("images").value,
-      false
-    );
-  });
-}
-addingEventListenerToLoadImageButton();
-
 function appendResponseCallback(data) {
-  document.getElementById("image").innerHTML = data;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities